packages/scripts/utils/string.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Given a string, returns a new string with dash separators converted to
+ * camelCase equivalent. This is not as aggressive as `_.camelCase` in
+ * converting to uppercase, where Lodash will also capitalize letters
+ * following numbers.
+ *
+ * @param {string} string Input dash-delimited string.
+ *
+ * @return {string} Camel-cased string.
+ */
+function camelCaseDash( string ) {
```

Comment:
If we're building something new, I'm wondering if we couldn't use this as an opportunity to address the fact that this function is insufficient in representing our capitalization standards, resulting in global scoped variables like `wp.escapeHtml` which should be corrected as `wp.escapeHTML`.

https://wordpress.slack.com/archives/C5UNMSU4R/p1542385406412600
https://wordpress.slack.com/archives/C5UNMSU4R/p1542722527426400

Reply:
I extracted it from the existing webpack config. It's still "private" but I'm happy to iterate on it and maybe even put it into a separate package.